install/install_for_windows.ps1

```diff
@@ -1,6 +1,4 @@
-﻿﻿#!/usr/bin/pwsh
-
-$INSTALL_FOLDER="C:\Windows"
+﻿$INSTALL_FOLDER="C:\Windows"
```

Comment:
@amri04 @Lambourl This is really questionable, isn't that a bit extreme ? 
Installing in his own "Program Files" folder would be really better, and we could use powershell to modify the path to our location : 

```
$oldpath = (Get-ItemProperty -Path 'Registry::HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Session Manager\Environment' -Name PATH).path
$newpath = “$oldpath;c:\path\to\folder”
Set-ItemProperty -Path 'Registry::HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Session Manager\Environment' -Name PATH -Value $newpath
```

If you don't see any problems regarding this, I'll create a new issue.

Reply:
A meaning about that @amri04  ? 😆

Reply:
Ok solved anyway now we install in `C:\ProgramData`